app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+
```

Comment:
there is no need for a blank line between the class definitions and class attributes.